common/pubspec.yaml

```diff
@@ -0,0 +1,9 @@
+name: realm_common
```

Comment:
we should add a README.md, CHANGELOG.md and LICENSE files to this package. For the LICENSE files we do a symlink to the LICENSE file in the root of the repo.